### DISCUSSION TITLE
Render Stream Continuously?

### DISCUSSION THREAD

**Discussion by Jafo232** [livewire/livewire#6276] [1. Help]
I have been tinkering with the wire:stream directive, with chatGPT similar to the demonstration at this year's Laracon.  Is there any way to get the content to render as it comes in in HTML?  I have found that it doesn't fully render until the stream has completed.  

![image](https://github.com/livewire/livewire/assets/1456061/27db59be-02be-442b-a59f-e45f99446e5a)

![image](https://github.com/livewire/livewire/assets/1456061/621fea76-ce00-478a-9047-e48d905ee558)

**Comment by keatliang2005** [comment]
form demo laracon

```php
// prop
public string $content;
// action
public function getContentThatStream()
{
    // stream content callback that uses $this->stream() method to perform obflush
    $this->content = $this->stream('content', $streamedContent);
}
```

```blade
<div wire:stream='content'>
    {{ $content }}
</div>
```
